packages/components/src/utils/colors-values.js

```diff
@@ -1,181 +1,86 @@
-/**
- * External dependencies
- */
-import { merge } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { rgba } from './colors';
 
-const BASE = {
-	black: '#000',
-	white: '#fff',
-};
+const white = '#fff';
 
-const G2 = {
```

Comment:
Good that we're getting rid of "g2" references, as they don't make much sense anymore!